.scripts/create_upgrade.php

```diff
@@ -75,14 +66,44 @@
 
 $upgrade_code = <<<___UPGRADE
 <?php
```

Comment:
We're in fact not loading the engine, and I think it might currently cause more trouble than benefits. So no access to views ATM.